src/components/error_page.jsx

```diff
@@ -0,0 +1,15 @@
+import { useRouteError } from 'react-router-dom';
+
+export default function ErrorPage() {
+  const error = useRouteError();
+
+  return (
+    <div id="error-page">
+      <h1>Oops!</h1>
+      <p>Sorry, an unexpected error has occurred.</p>
```

Comment:
- [ ] In the remove-unit section:


![image](https://github.com/Elerqsousy/grandeur-frontend/assets/57421823/567ce243-f2c7-4737-8dba-ba645fe30200)

If this feature is not being right now implemented, kindly say something like "Under construction", because if you display an error it might cause confusion. 👍